### PR TITLE
minor fix for time range

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

File: scripts/visualize/timeDayUse.R [hunk 1]
```diff
@@ -57,7 +57,7 @@ visualize.timeDayUse_app <- function(viz=as.viz("timeDayUse_app")) {
     
     barplot(hourSum$n, names.arg = hourSum$hour,
             xlab = "Hour of Day",ylab = "Sessions",
-            main = paste("Frome",paste(range_days,collapse = " to ")),
```
Comment (ldecicco-USGS) [inline scripts/visualize/timeDayUse.R]: alternative spellings

**Comment by jread-usgs** [comment]
hold merge, looks like these are reversed for all

**Comment by ldecicco-USGS** [comment]
oh no

**Comment by jread-usgs** [comment]
I got a change - one min

**Pull request merged**